src/main/java/Type.java

```diff
@@ -0,0 +1,6 @@
+/**
+ * Enumeration representing the type of task object.
+ */
+public enum Type {
+    TODO, DEADLINE, EVENT
```

Comment:
oh good idea making the types into enums!

Reply:
Oh one more thing I want to add, you should try to improve the way you make commit messages. Start with capital letter, using imperatives as the first word, etc.

Reply:
OKay!! I haven't really paid much attention to my commit messages so far HAHA! Thanks for pointing it out!